spec/factories/state_file_nj_intakes.rb

```diff
@@ -0,0 +1,111 @@
+#  last_sign_in_at                   :datetime
+#  last_sign_in_ip                   :inet
+#  locale                            :string           default("en")
+#  locked_at                         :datetime
+#  message_tracker                   :jsonb
+#  payment_or_deposit_type           :integer          default("unfilled"), not null
+#  permanent_apartment               :string
+#  permanent_city                    :string
+#  permanent_street                  :string
+#  permanent_zip                     :string
+#  phone_number                      :string
+#  phone_number_verified_at          :datetime
+#  primary_birth_date                :date
+#  primary_esigned                   :integer          default("unfilled"), not null
+#  primary_esigned_at                :datetime
+#  primary_first_name                :string
+#  primary_last_name                 :string
+#  primary_middle_initial            :string
+#  primary_signature                 :string
+#  primary_ssn                       :string
+#  primary_suffix                    :string
+#  raw_direct_file_data              :text
+#  referrer                          :string
+#  routing_number                    :string
+#  sign_in_count                     :integer          default(0), not null
+#  source                            :string
+#  spouse_birth_date                 :date
+#  spouse_esigned                    :integer          default("unfilled"), not null
+#  spouse_esigned_at                 :datetime
+#  spouse_first_name                 :string
+#  spouse_last_name                  :string
+#  spouse_middle_initial             :string
+#  spouse_ssn                        :string
+#  spouse_suffix                     :string
+#  tax_return_year                   :integer
+#  unfinished_intake_ids             :text             default([]), is an Array
+#  unsubscribed_from_email           :boolean          default(FALSE), not null
+#  withdraw_amount                   :integer
+#  created_at                        :datetime         not null
+#  updated_at                        :datetime         not null
+#  federal_submission_id             :string
+#  primary_state_id_id               :bigint
+#  spouse_state_id_id                :bigint
+#  visitor_id                        :string
+#
+# Indexes
+#
+#  index_state_file_nj_intakes_on_email_address        (email_address)
+#  index_state_file_nj_intakes_on_hashed_ssn           (hashed_ssn)
+#  index_state_file_nj_intakes_on_primary_state_id_id  (primary_state_id_id)
+#  index_state_file_nj_intakes_on_spouse_state_id_id   (spouse_state_id_id)
+#
+FactoryBot.define do
+  factory :state_file_nj_intake do
+    transient do
+      filing_status { 'single' }
+    end
+
+    eligibility_lived_in_state { "yes" }
+    eligibility_out_of_state_income { "no" }
+    raw_direct_file_data { File.read(Rails.root.join('app', 'controllers', 'state_file', 'questions', 'df_return_sample.xml')) }
```

Comment:
[PEBBLE] Do you want to use one of the NJ DF return xmls you created as the baseline DF XML for this factory?